.github/workflows/openssf.yaml

```diff
@@ -0,0 +1,58 @@
+name: Scorecard Supply-chain Security
+on:
+  schedule:
+    - cron: '32 21 * * 2'
```

Comment:
include a link to the `https://crontab.guru/` for this, I like doing so just like w/ `regex`s